dataikuapi/dss_plugin_mlflow/utils.py

```diff
@@ -6,7 +9,7 @@
 
 
 class MLflowHandle:
-    def __init__(self, client, project_key, managed_folder_name, host=None):
+    def __init__(self, client, project, managed_folder, host=None):
```

Comment:
It seems to me that it would be simpler if `managed_folder` could only be a handle on a MF, and not a MF name.
We would avoid checking that it exists, etc. In this case, we would only check that `managed_folder` is an instance of `DSSManagedFolder` and add its type to the docstring.